apps/src/lib/levelbuilder/CourseVersionPublishingEditor.jsx

```diff
@@ -0,0 +1,188 @@
+            </p>
+          </HelpTip>
+        </label>
+        <label>
+          Version Year
+          <select
+            value={this.props.versionYear}
+            style={styles.dropdown}
+            onChange={this.props.updateVersionYear}
+          >
+            <option value="">(None)</option>
+            {this.props.versionYearOptions.map(year => (
+              <option key={year} value={year}>
+                {year}
+              </option>
+            ))}
+          </select>
+        </label>
+        <label>
+          Published State
+          <select
+            className="publishedStateSelector"
+            value={this.props.publishedState}
+            style={styles.dropdown}
+            onChange={this.handlePublishedStateChange}
+          >
+            {publishedStates.map(state => (
+              <option key={state} value={state}>
+                {state}
+              </option>
+            ))}
+          </select>
+          <HelpTip>
+            <table>
+              <thead>
+                <tr>
+                  <th>Publish State</th>
+                  <th>Overview</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td style={styles.tableBorder}>Pilot</td>
+                  <td style={styles.tableBorder}>
+                    A limited set of teachers who are in a pilot experiment can
+                    see and assign the course.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Preview</td>
+                  <td style={styles.tableBorder}>
+                    Anyone who has the link can view the course and make
+                    progress on it. It is not assignable by teachers yet.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Assignable</td>
+                  <td style={styles.tableBorder}>
+                    The course is now a choice in the dropdown that is
+                    assignable but is not the recommended course.
+                  </td>
+                </tr>
+                <tr>
+                  <td style={styles.tableBorder}>Recommended</td>
+                  <td style={styles.tableBorder}>
+                    The course is the recommended course. It is assignable and
+                    we try to get teachers to use this course.
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </HelpTip>
+        </label>
+        {this.props.publishedState === 'Pilot' && (
+          <label>
+            Pilot Experiment
+            <HelpTip>
+              <p>
+                If specified, this course will only be accessible to
+                levelbuilders, or to classrooms whose teacher is enrolled in
+                this pilot experiment
+              </p>
+            </HelpTip>
+            <input
```

Comment:
when I try to type in this box I get the string `[object Object]`, not sure why

![Screen Shot 2021-05-17 at 2 56 15 PM](https://user-images.githubusercontent.com/46464143/118561634-21de8600-b720-11eb-8709-23cf222aa2df.png)